### PR TITLE
[TP-194578] Update logo on dev site

### PR DESCRIPTION
**Pull request by CourtneyWill**

### CODE REVIEW

**Comment by CourtneyWill** [comment]
https://civicplus.tpondemand.com/entity/194578-update-logo-on-api-docs-site

File: layouts/default.html [hunk 1]
```diff
@@ -19,7 +19,7 @@
     <div id="header-wrapper">
       <div id="header">
         <div>
-          <a class="logo" href="/"><img src="/images/logo.png" alt="SeeClickFix API" /></a>
+          <a class="logo" href="/"><img src="/images/seeclickfix-311-crm-logo.svg" alt="SeeClickFix API" height="33px"/></a>
           <ul class="nav">
```
Comment (CourtneyWill) [inline layouts/default.html:23]: While we are here should we consider a "better" `alt` text?
Is there a current organization-wide `alt` text for logos?

Comment (ssawyer1024) [reply layouts/default.html:23]: "SeeClickFix API" is definitely incorrect in this context. Even "SeeClickFix Home" or similar would be more accurate.

File: layouts/default.html [hunk 2]
```diff
@@ -19,7 +19,24 @@
     <div id="header-wrapper">
       <div id="header">
         <div>
-          <a class="logo" href="/"><img src="/images/logo.png" alt="SeeClickFix API" /></a>
+          <a class="logo" href="/">
+            <svg height="33" viewBox="0 0 220 54" fill="none" role="img" xmlns="http://www.w3.org/2000/svg" aria-label="SeeClickFix Logo">
```
Comment (CourtneyWill) [inline layouts/default.html:23]: Any thoughts or feedback on the `aria-label`: "SeeClickFix Logo"

**Review by Gaurav0 — COMMENTED** [review]
Why are we inlining the logo's source code in the html?

File: layouts/default.html (hunk 2 above)
Comment (Gaurav0) [reply layouts/default.html:23]: I would have used the <img tag and the `alt` attribute

I would use the text in the image, "CivicPlus SeeClickFix 311 CRM"

**Review by Gaurav0 — APPROVED** [review]

File: layouts/default.html [hunk 3]
```diff
@@ -19,7 +19,9 @@
     <div id="header-wrapper">
       <div id="header">
         <div>
-          <a class="logo" href="/"><img src="/images/logo.png" alt="SeeClickFix API" /></a>
+          <a class="logo" href="/">
+            <img src="/images/seeclickfix-311-crm-logo.png" height="33px" alt="CivicPlus SeeClickFix 311 CRM" />
```
Comment (zbeat) [inline layouts/default.html:23]: @CourtneyWill I see the actual image size is 440w x 108h.  Was that an intentional asset size choice, for the 33px rendered height?

Comment (CourtneyWill) [reply layouts/default.html:23]: @zbeat I don't see an option to export a custom dimension. There is only the preset option [1x, 1.5x, 2x, 3x.....].
I exported the file and set the size in HTML.

**Pull request merged**